### ISSUE TITLE
Import/export settings between OS installations

### ISSUE THREAD

**Issue by JosepBalague** [MrAlex94/Waterfox#2239]
**Is your feature request related to a problem? Please describe.**
I spend 2 hours configurating WF after a new clean Windows installation. Not sure if copying C:\Users\your_old_name\AppData\Roaming\Waterfox\Profiles\aBunchOfCharacters.default is enough to export configuration between OS installations.

**Describe the solution you'd like**
Export current config (right sandwich/options), addons, address bar in a file.
Import this file in a new WF.
WF should generate a file similar VS does (Exported-2021-09-08.vssettings)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by ghost** [comment]
Profile is enough I think.

Of course if name of profile is changed you need to update ini file.

**Comment by MrAlex94** [comment]
Waterfox now includes an option to import from specific folder - File > Import from another browser. So drop the profile or have it somewhere accessible  where the file picker can find it.

**Comment by JosepBalague** [comment]
> 
> 
> Profile is enough I think.
> 
> Of course if name of profile is changed you need to update ini file.

@hardhub,

Thanks. I followed these steps (I don't remember were I read them) and a full back up is done correctly:

1. On the old computer, open Waterfox, type "about:support". In the first block, click on the button "Profile Folder –> open the folder".
2. Quit Waterfox.
3.     Look at the folder you just opened; get up one level: this is your profile folder. The path to the folder will be something like: C:\Users\<user>AppData\Roaming\Waterfox\Profiles\aBunchOfCharacters.default
4. Copy that folder on a medium that you'll carry to the new computer (ex.: USB key)
5. On the new computer, install Waterfox and run it once. It will create a profile
6. Do similar manoeuvres to find your profile on your new computer. Its path will be something like: C:\Users\<user>AppData\Roaming\Waterfox\Profiles\anotherBunchOfCharacters.default
7. Quit Waterfox
8.     Erase the content of the new profile folder and replace it with a copy of the content of your old profile folder (the one you carried over on your USB key).

These steps worked fine. for a full config WF backup (plus bookmarks, add-ons, passwords, master password)

@MrAlex94,

I tested File > Import from another browser. > Firefox. In last step I pressed X for _cancel_ process. Result: all add-ons, bookmarks, passwords, master password and other config removed. Fortunately I had a backup of BunchOfCharacters.default folder.
